### DISCUSSION TITLE
Welcome to attendancesystem_flutter Discussions!

### DISCUSSION THREAD

**Discussion by merthankavak** [merthankavak/attendancesystem_flutter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ArchieTzy** [comment]
Hello, I'm Archie Raven M. Raga from Philippines and I would like to ask on how to fix service_helper.dart in your _utility folder? Thanks in advance :)
